ground/src/main/res/layout/basemap_layout.xml

```diff
@@ -43,6 +46,19 @@
       android:fitsSystemWindows="true"
       tools:background="@color/colorGrey800">
 
+      <ImageView
```

Comment:
For the new "drop a pin" task, we should use the pin icon instead of the cross hairs. Cross hairs are probably best avoided due to their negative connotations.

Reply:
Done